app/blinken.js

```diff
@@ -46,8 +46,12 @@
     var self = this;
     $.getJSON(environment_url + "/cgi-bin/icinga/status.cgi?servicestatustypes=20&jsonoutput=1", function(data) {
       var service_status = data.status.service_status;
-      var critical_entries = service_status.filter(function(o) { return o.status === "CRITICAL" && o.has_been_acknowledged === false }).length;
-      var warning_entries = service_status.filter(function(o) { return o.status === "WARNING" && o.has_been_acknowledged === false }).length;
+      var critical_entries = service_status.filter(function(o) { return o.status === "CRITICAL" &&
+                                                                        o.has_been_acknowledged === false &&
+                                                                        o.in_scheduled_downtime === false }).length;
+      var warning_entries = service_status.filter(function(o) { return o.status === "WARNING" &&
+                                                                       o.has_been_acknowledged === false &&
+                                                                       o.in_scheduled_downtime === false }).length;
```

Comment:
You could avoid some duplication here by filtering the `service_status` array, prior to pulling out the warnings and criticals.

Reply:
Good idea. I'll do this.